pubsub/src/main/java/com/github/twitch4j/pubsub/domain/SupportActivityFeedData.java

```diff
@@ -0,0 +1,58 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.github.twitch4j.common.annotation.Unofficial;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.OptionalInt;
+
+@Unofficial
+@Data
+@Setter(AccessLevel.NONE)
+@NoArgsConstructor
+public class SupportActivityFeedData {
+    private String channelId;
+    private String eventId;
+    private String userId;
+    private String userLogin;
+    private String userDisplayName;
+    private String userProfileImageUrl;
+    private String action;
+    private String source;
+    private Integer quantity;
+
+    @Unofficial
+    public boolean isCheer() {
+        return "BITS".equals(source) || "CHEER".equals(action);
```

Comment:
```suggestion
    public boolean isBitsCheer() {
        return "BITS".equals(source) && "CHEER".equals(action);
```
in case we need to add [`isDirectCheer`](https://github.com/twitch4j/twitch4j/pull/566) down the line